client/ayon_substancedesigner/api/project_creation.py

```diff
@@ -147,35 +161,60 @@ def create_project_with_from_template(project_settings=None):
                 template_filepath = get_template_filename_from_project(
                     resources_dir, project_template
                 )
+        elif project_template_setting["template_type"] == (
+            "custom_template"
+            ):
+                custom_template = project_template_setting["custom_template"]
+                project_template = custom_template["custom_template_graph"]
+                if not project_template:
+                    log.warning("Project template not filled. "
+                                "Skipping project creation.")
+                    continue
+
+                path = custom_template["custom_template_path"]
+                if not path:
+                    log.warning("Template path not filled. "
+                                "Skipping project creation.")
+                    continue
+                folder_entity, task_entity = get_entity_data(project_name)
+                template_filepath = resolve_template_path(
+                    path, project_name, folder_entity, task_entity
+                )
+                if not os.path.exists(template_filepath):
+                    log.warning("Template path does not exist yet. "
```

Comment:
Should this report which file was not found?
```suggestion
                    log.warning(f"Template path '{template_filepath}' does not exist yet. "
```